src/app/TideMatchSet.cpp

```diff
@@ -21,11 +21,11 @@ char TideMatchSet::match_collection_loc_[] = {0};
 char TideMatchSet::decoy_match_collection_loc_[] = {0};
 
 TideMatchSet::TideMatchSet(Arr* matches, double max_mz)
-  : matches_(matches), max_mz_(max_mz), exact_pval_search_(false), elution_window_(0) {
+  : matches_(matches), max_mz_(max_mz), exact_pval_search_(false), elution_window_(0), cur_score_function_(XCORR_SCORE) {
 }
 
 TideMatchSet::TideMatchSet(Peptide* peptide, double max_mz)
-  : peptide_(peptide), max_mz_(max_mz), exact_pval_search_(false), elution_window_(0) {
+  : peptide_(peptide), max_mz_(max_mz), exact_pval_search_(false), elution_window_(0), cur_score_function_(XCORR_SCORE) {
```

Comment:
Does this break the p-value code? Or any part of the code that doesn't use the score function xcorr_score?

Reply:
No, it's not. In tide-search, it will be overwritten